dali/base/dautils.cpp

```diff
@@ -515,7 +537,12 @@ void CDfsLogicalFileName::set(const char *name, bool removeForeign)
         lfn.set(full);
         return;
     }
-    normalizeName(name, lfn, false);
+    external = memicmp(name,EXTERNAL_SCOPE "::",sizeof(EXTERNAL_SCOPE "::")-1)==0;
+    if (external && checkEp(name))
+        // TODO Should check the name is a valid OS filename
+        lfn.set(name);
+    else
```

Comment:
if checkEp failed then prior logic did _not_ set external.. but this leaves set.

It would be cleaner if the above code + comment was in an e.g. isExternal function and this looked like:
```
if (isExternal(name)) 
{
    lfn.set(name);
    external = true;
}
else
    normalizeName(name, lfn, false);
```